packages/swatch/swatch-group.md

```diff
@@ -217,6 +233,27 @@ When available, the value of `selected` will be constructed from the `value` att
 <div>Selected: [ "color-2", "color-1", "color-3" ]</div>
 ```
 
-## Swatch modifying attributes
+### Swatch modifying attributes
+
+An `<sp-swatch-group>` element can be modified by the following attributes/properties to customize its delivery as desired for your use case: `border`, `disabled`, `mixedValue` (accepted as the `mixed-value` attribute), `nothing`, `rounding`, `shape`, and `size`. Use these in concert with each other for a variety of final visual deliveries. Applying a value for one of these attributes/properties to an `<sp-swatch-group>` element will have it forward the value to all of the `<sp-swatch>` elements that are a direct child of the group, overriding any value that may be applied directly to those children. For example:
```

Comment:
Does SWC support all of these attributes on the swatch group? Maybe I'm adding it wrong, but if I put `disabled` on the sp-swatch-group nothing happens. Same with mixed value and nothing- I don't see stories for those use cases either. 

Disabled, nothing, and mixedValue all work on individual swatches, but _not_ for swatch group, so maybe we leave those off?